lib/core/src/lib/viewer/components/viewer.component.spec.ts

```diff
@@ -115,6 +115,37 @@ describe('ViewerComponent', () => {
 
     });
 
+    describe('originalMimeType', () => {
+
+        it('should set alt attribute to originalMimeType when originalMimeType is provided', () => {
+            component.originalMimeType = 'image/png';
+            fixture.detectChanges();
+            const altAttribute: string = fixture.nativeElement.querySelector('.adf-viewer__mimeicon').getAttribute('alt');
```

Comment:
Could you please move this part: fixture.nativeElement.querySelector('.adf-viewer__mimeicon') to function because it is used many times now?